<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeMain">
     <column>
      <property name="text">
       <string>index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Año</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipster</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Importe</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblConjunta">
     <property name="text">
      <string>Conjuntas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeConjunta">
     <column>
      <property name="text">
       <string>index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Año</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Importe</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
